### ISSUE TITLE
Testing brainstorming

### ISSUE THREAD

**Issue by nikipapadatou** [ethersphere/bee#4480]
The following document focuses on a draft approach regarding the testing strategy. Each team of the organisation (where relevant) is called to brainstorm, discuss and get the appropriate tasks. Also, they are called to suggest improvements for the other teams whenever there is a dependency. 

https://hackmd.io/VVLu7xGxSR6nkv-YGvu0xA

This is a placeholder task for the bee team, where all the members are called to express their opinions on what we should focus on the tests. Feel free to create a replicate document and write there your comments, improvements, ideas and the relevant tasks.

Please, attach your documents in the comments.

**Comment by ldeffenb** [comment]
> https://hackmd.io/@ethswarm/rkiiXGEbT

This might be ok in bee-backlog, but the main bee repository issues?

![image](https://github.com/ethersphere/bee/assets/5175026/00a3bebc-dacd-42db-a9ad-3a2c2086a7fe)

**Comment by tmm360** [comment]
I agree, please don't use references to private contents on public repos.
Actually, I would like to access documents also in the backlog area, anyway...
